instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/patches/client.rb

```diff
@@ -53,9 +53,7 @@ module Client
           def query(sql, options = {})
             tracer.in_span(
               database_span_name(sql),
-              attributes: client_attributes.merge(
-                'db.statement' => obfuscate_sql(sql)
-              ),
+              attributes: %I[include obfuscate].include?(config[:db_statement]) ? client_attributes.merge('db.statement' => obfuscate_sql(sql)) : client_attributes,
```

Comment:
💭 It looks like we are evaluating this configuration every time we execute this query. 

How would we feel about memoizing this value or evaluating this at configuration time?

Reply:
I will be an exercise for me to memoize and run a simple benchmark on both approaches 👍